cli/cmd/install.go

```diff
@@ -141,6 +148,8 @@ const (
 	defaultIdentityTrustDomain        = "cluster.local"
 	defaultIdentityIssuanceLifetime   = 24 * time.Hour
 	defaultIdentityClockSkewAllowance = 20 * time.Second
+
+	caCommonName = "ca.linkerd.cluster.local"
```

Comment:
curious about this name, should it be namespace-specific? also is there a reason this is different from most `*.svc.cluster.local` names, or is this convention?

Reply:
Yeah, I didn't put much thought into this name. LMK what's most appropriate. I basically follow the identity issuer's name [convention](https://github.com/linkerd/linkerd2/blob/5ece3430eb72d8286214e599997f3f703f943fd3/cli/cmd/install.go#L875); i.e. without the `.svc` (which makes sense, since there isn't a `Service` with this name).

Reply:
I think following the identity issuer's name makes sense. Note that the name varies by namespace.